jms/src/test/scala/akka/stream/alpakka/jms/scaladsl/JmsAckConnectorsSpec.scala

```diff
@@ -364,7 +366,7 @@ class JmsAckConnectorsSpec extends JmsSpec {
     }
 
     "shutdown when waiting to acknowledge messages" in withServer() { server =>
-      val connectionFactory = server.createConnectionFactory
+      val connectionFactory = server.createQueueConnectionFactory
```

Comment:
What's the purpose of this change?

Reply:
I honestly think i was trying to change that in the test i introduced, and just missed 😓